tests/countries/test_poland.py

```diff
@@ -353,3 +353,41 @@ def test_l10n_uk(self):
             ("2022-12-25", "Різдво Христове"),
             ("2022-12-26", "Другий день Різдва"),
         )
+
+    def test_l10n_de(self):
```

Comment:
This needs to be placed before `test_l10n_en_us()` (to keep alphabetical order of languages).